### ISSUE TITLE
indent-blankline not calling setup

### ISSUE THREAD

**Issue by MarkusLohmayer** [nix-community/nixvim#647]
**Plugin affected**: indent-blankline
**Nixpkgs channel**: unstable

## Description

I tried using the indent-blankline plugin but it did not work.
Looking at the file
https://github.com/nix-community/nixvim/blob/main/plugins/utils/indent-blankline.nix
I see that `require("ibl").setup()` is not called, see
https://github.com/lukas-reineke/indent-blankline.nvim

**Comment by jonboh** [comment]
You can run it manually:
```nix
  config = {
    plugins.indent-blankline =
      {
        enable = true;
      };
    options = {
      list = true;
      listchars = "space:⋅";
    };
    extraConfigLua = ''
      require("ibl").setup()
    '';
  };
```
With that config I'm getting the default behavior of ident-blankline version 3.

It seems that when the ident-blankline plugin was introduced into nixvim, version 3 of the plugin was still not released, that's why `require("ibl").setup()` is not called. 
Here is the [migration guide](https://github.com/lukas-reineke/indent-blankline.nvim/wiki/Migrate-to-version-3), the options in nixvim should probably be updated.

**Comment by GaetanLepage** [comment]
We have planned to update to the latest version: https://github.com/nix-community/nixvim/pull/650

**Comment by GaetanLepage** [comment]
Should be fixed as #650 has been merged.
Please, re-open this issue if some problems remain.

**Comment by MarkusLohmayer** [comment]
Thanks! :)
